app/interfaces/api/v2/mi/scheme_ten.rb

```diff
@@ -0,0 +1,44 @@
+module API
+  module V2
+    module MI
+      class SchemeTen < Grape::API
```

Comment:
💭 Consider renaming the file/class to something like `API::V2::MI::AgfsSchemeTenClaims`, first because the resource appears to be _claims_ and second because _scheme ten_ seems to apply only specifically to *AGFS*